cypress/e2e/supervision/finance/finance-tab.cy.js

```diff
@@ -0,0 +1,201 @@
+ describe(
+  "Finance tab",
+  { tags: ["@supervision", "@supervision-regression", "@finance-tab", "@finance"] },
+  () => {
+
+    beforeEach(() => {
+      cy.loginAs("Finance User Testing");
+      cy.createClient()
+        .withOrder()
+        .withSupervisionLevel()
+        .withOrderStatus();
+      cy.get("@client").then(({ caseRecNumber }) => {
+        cy.assignSOPNumberToClient(caseRecNumber)
+      });
+      cy.get("@client").then(({ id }) => {
+        cy.visit(`/supervision/#/clients/${id}`);
+      });
+      cy.get('.TABS_FINANCEINFO').click();
+    });
+
+    it("allows me to add credit and shows this correctly in the finance invoice list", () => {
```

Comment:
Can this be reworded to remove the pronoun? Something like "can add credit and display it in the finance invoice list".